dbms/src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -135,9 +135,6 @@ InterpreterSelectQuery::InterpreterSelectQuery(
     , input(input_)
     , log(&Logger::get("InterpreterSelectQuery"))
 {
-    if (!context.hasQueryContext())
```

Comment:
Here is the cause of the problem, all queries without query_context and containing `in clauses` will crash. Because it's exposing the internal context.